webviz_subsurface/_abbreviations/reservoir_simulation.py

```diff
@@ -0,0 +1,71 @@
+import json
+import pathlib
+import warnings
+
+
+_DATA_PATH = pathlib.Path(__file__).parent.absolute() / "abbreviation_data"
+
+SIMULATION_VECTOR_TERMINOLOGY = json.loads(
+    (_DATA_PATH / "reservoir_simulation_vectors.json").read_text()
+)
+
+RESERVOIR_SIMULATION_UNIT_TERMINOLOGY = json.loads(
+    (_DATA_PATH / "reservoir_simulation_unit_terminology.json").read_text()
+)
+
+
+def simulation_unit_reformat(ecl_unit: str, unit_set: str = "METRIC"):
+    """Returns the simulation unit in a different, more human friendly, format if possible,
+    otherwise returns the simulation unit.
+    * `ecl_unit`: Reservoir simulation vector unit to reformat
+    * `unit_set`: Currently only valid option is the default "METRIC" (defined as in Eclipse E100)
+    """
+    try:
+        unit = RESERVOIR_SIMULATION_UNIT_TERMINOLOGY[unit_set][ecl_unit]
+    except KeyError:
+        unit = ""
+
+    return unit
+
+
+def simulation_vector_base(vector: str):
+    """Returns base name of simulation vector
+    E.g. WOPR for WOPR:OP_1 and ROIP for ROIP_REG:1
+    """
+
+    return vector.split(":", 1)[0].split("_", 1)[0]
+
+
+def simulation_vector_description(vector: str):
+    """Returns a more human friendly description of the simulation vector if possible,
+     otherwise returns the input as is.
+    """
+    [vector_name, node] = vector.split(":", 1) if ":" in vector else [vector, None]
+    if len(vector_name) == 8:
```

Comment:
Found one weakness here, need to verify that metadata type is "region", other vectors may have 8 signs too, will make a fix